trust/signature.go

```diff
@@ -37,7 +39,7 @@ func (DummySigner) Sign(key crypto.PrivateKey, data []byte) ([]byte, error) {
 	sig := make([]byte, n)
 
 	if _, err := rand.Read(sig); err != nil {
-		return nil, fmt.Errorf("sign: %w", err)
+		return nil, fmt.Errorf("rand.DummySigner.Sign: %w: %s", ErrSigning, err)
```

Comment:
Not sure if `rand.DummySigner.Sign` is a good idea here. I assume `rand` is the package name (should be `trust` here then) but what if a user changes the import name of that package? Then `pkg.Struct.Method` might become confusing as the user imported it under a different name. I'd personally omit the pkg part and just name it `DummySigner.Sign` instead. What do you think?